cmd/dmsgpost/dmsgpost.go

```diff
@@ -0,0 +1,226 @@
+// package main cmd/dmsgpost/dmsgpost.go
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	cc "github.com/ivanpirog/coloredcobra"
+	"github.com/skycoin/skywire-utilities/pkg/buildinfo"
+	"github.com/skycoin/skywire-utilities/pkg/cipher"
+	"github.com/skycoin/skywire-utilities/pkg/cmdutil"
+	"github.com/skycoin/skywire-utilities/pkg/logging"
+	"github.com/skycoin/skywire-utilities/pkg/skyenv"
+	"github.com/spf13/cobra"
+
+	"github.com/skycoin/dmsg/pkg/disc"
+	dmsg "github.com/skycoin/dmsg/pkg/dmsg"
+	"github.com/skycoin/dmsg/pkg/dmsghttp"
+)
+
+var (
+	dmsgDisc     string
+	dmsgSessions int
+	dmsgpostData string
+	//	dmsgpostHeader string
+	sk            cipher.SecKey
+	dmsgpostLog   *logging.Logger
+	dmsgpostAgent string
+	logLvl        string
+)
+
+func init() {
+	rootCmd.Flags().StringVarP(&dmsgDisc, "dmsg-disc", "c", "", "dmsg discovery url default:\n"+skyenv.DmsgDiscAddr)
+	rootCmd.Flags().IntVarP(&dmsgSessions, "sess", "e", 1, "number of dmsg servers to connect to")
```

Comment:
This is only used for making the request, i.e. the client, so why would we want to have multiple sessions? We dont seem to have any logic for multiplexing anyways. So seems useless for now.